src/plugins/intel_gna/src/backend/gna_limitations.cpp

```diff
@@ -155,6 +155,13 @@ bool Validator_30::ValidateCnn2D(const std::string &name, const uint32_t inHeigh
     return ValidationSuccesful(throwOnError, error, name, "Convolution2D");
 }
 
+bool Validator_30::ValidateCnn1D(const std::string& name, const uint32_t inHeight, const uint32_t inWidth,
+    const uint32_t inChannels, const uint32_t kH, const uint32_t kW, const uint32_t kN,
+    const uint32_t strideH, const uint32_t strideW, const uint32_t dilationH, const uint32_t dilationW,
+    OvGnaType inPrecision, bool exception) const {
+        return false;
```

Comment:
Why it should return false by default?

Reply:
In simple words validator for 1D convolution on GNA3.0 and previous is not implemented by AbstractValidator interface. My intention was to not touching 1Dconvolution path for tergets<3.5